datasetIngestor/ingestDataset.go

```diff
@@ -176,7 +176,7 @@ func createOrigDatablocks(client *http.Client, APIServer string, fullFileArray [
 
 	if totalFiles > TOTAL_MAXFILES {
 		return fmt.Errorf(
-			"dataset exceeds (%v) the maximum number of files per dataset , which can currently be handled by the archiving system (%v)",
+			"dataset exceeds (%v) the maximum number of files per dataset , which cannot currently be handled by the archiving system (%v)",
```

Comment:
I think it was correct before

Reply:
I think it's a poorly written message because the original version has a double meaning: "exceeds the maximum number of files that can otherwise be handled by the archival system" (not true) and "exceeds the maximum number of files that can be handled by the archival system" (true).

I'll rewrite this again.